pubspec.yaml

```diff
@@ -29,6 +29,16 @@ dependencies:
   authentication_view: ^0.4.1
   geolocator: '^5.1.1'
 
+  # Used to store the data such as access token/other sensitive information securely.
+  flutter_secure_storage: ^3.2.1+1
+
+  # Interceptor is used to modify API Request by adding auth header in each request.
+  http_interceptor: any
```

Comment:
Ideally should be version specific. Using ```any``` will fetch latest version and it might not be updated some time.

Reply:
Fixed.